utils/fetchIssuedAssets.ts

```diff
@@ -0,0 +1,60 @@
+import ERC725js, { ERC725JSONSchema } from '@erc725/erc725.js';
+import LSP12Schemas from '@erc725/erc725.js/schemas/LSP12IssuedAssets.json';
+import LSP4Schemas from '@erc725/erc725.js/schemas/LSP4DigitalAsset.json';
+
+const verifyCreatorShip = async (
+  ownerAddress: string,
+  provider: any,
+  issuedAssetAddress: string,
+): Promise<string> => {
```

Comment:
So... a function called `verifyCreatorShip` return a string? What is that string?

Maybe you want to convert this function so it return a boolean true/false if the owner is also the creator?